modules/wallet/database.go

```diff
@@ -206,6 +207,39 @@ func dbDeleteSpentOutput(tx *bolt.Tx, id types.OutputID) error {
 	return dbDelete(tx.Bucket(bucketSpentOutputs), id)
 }
 
+// dbReinitialize wipes and reinitializes a wallet database.
+func dbReinitialize(tx *bolt.Tx) error {
+	for _, bucket := range dbBuckets {
+		err := tx.DeleteBucket(bucket)
+		if err != nil {
+			return err
+		}
+		_, err = tx.CreateBucket(bucket)
+		if err != nil {
+			return err
+		}
+	}
```

Comment:
Ok, so doing one reinit led to a filesize increase of `wallet.db` from 815MB -> 898MB